homeassistant/components/opentherm_gw/__init__.py

```diff
@@ -233,6 +245,20 @@ async def reset_gateway(call):
         DOMAIN, SERVICE_RESET_GATEWAY, reset_gateway, service_reset_schema
     )
 
+    async def set_ch_ovrd(call):
+        """Set the central heating override on the OpenTherm Gateway."""
+        gw_dev = hass.data[DATA_OPENTHERM_GW][DATA_GATEWAYS][call.data[ATTR_GW_ID]]
+        hass.async_create_task(
```

Comment:
```suggestion
        await self.hass.async_add_executor_job(
```

Reply:
Shouldn't this run in the executor?

Reply:
OK, so I finally got to test this today. Whenever I try to use async_add_executor_job it throws a `TypeError: coroutines cannot be used with run_in_executor()`. Looks like we're better of not making this change.

Reply:
Sorry, I didn't realize this was async.  I'll update the suggestion.